src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -100,6 +100,26 @@ public static PartyDetails createDwpPartyDetails() {
 
     public static PartyDetails createJointPartyDetails(SscsCaseData caseData) {
```

Comment:
Is this method called from anywhere?

Reply:
Yes , it's called in method buildHearingPartiesDetails() in same class